src/model/FeaturesView.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { Fragment, useCallback } from 'react';
+import ProtvistaTrack from 'protvista-track';
+import ProtvistaManager from 'protvista-manager';
+import ProtvistaDatatable from 'protvista-datatable';
+import ProtvistaSequence from 'protvista-sequence';
+import ProtvistaNavigation from 'protvista-navigation';
+import { loadWebComponent } from '../utils/utils';
+
+type FeatureLocation = {
+  value: number;
+  modifier: string;
+};
+
+type Feature = {
+  type: string;
```

Comment:
These data will will come from a backend request so best to be paranoid and make them optional types.

Reply:
Had a chat with Jie - a feature will always have a type and a location, although there are 3 different modifiers for the position (added an enum) and if the modifier is of a certain value we can't assume the FeatureLocation will have a value, so now checking for that.